### ISSUE TITLE
No 2FA for GitHub bot is mandatory ? 

### ISSUE THREAD

**Issue by freak4pc** [fastlane/ci#648]
Going through the onboarding process, It seems that when creating a bot it mustn't have 2FA enabled.

<img width="636" alt="f1d910c1-a66f-4871-b0fe-6c8d3845f231" src="https://user-images.githubusercontent.com/605076/38663392-563589f6-3e3f-11e8-878b-d168f2055e48.png">

This is an issue for organizations that enforce 2FA:

> Two-factor authentication can't be disabled because you belong to an organization that requires it. Visit your organization settings page to review which organizations require 2FA.

I tried looking through milestone issues but couldn't find anything related to this :) 

Thanks!

**Comment by freak4pc** [comment]
Might be possible using Personal Access Token but I actually didn't get a chance to try

**Comment by freak4pc** [comment]
I’ll test and confirm. Would you like me to push a PR altering the note to add PAT as an option ?

**Comment by minuscorp** [comment]
I have to say that my _bot account_ is my own GitHub account, I have two factor authentication active and that haven't make me go through any trouble. Tokens are independent of the authentication used as far as it seems

**Comment by freak4pc** [comment]
Yeah but the onboarding notice is misleading (I abandoned onboarding at that point)

**Comment by taquitos** [comment]
@freak4pc that would be awesome if you submitted a pr for that change 👍

**Comment by freak4pc** [comment]
You got it 🤘

**Comment by taquitos** [comment]
@freak4pc looks like @AndrewMcBurney got to it already
